app/javascript/components/event/EventEdit.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react';
+import Axios from 'axios';
+
+export default class EventEdit extends React.Component {
```

Comment:
this component looks similar to previous. are their any opportunities to join these 2 components?